app/Http/Controllers/CompetitionsController.php

```diff
@@ -37,18 +37,6 @@ public function __construct(UserRepositoryContract $repository, Manager $manager
     }
 
     /**
-     * Display a listing of the resource.
-     *
-     * @return \Illuminate\Http\Response
-     */
-    public function index()
```

Comment:
I thought we were keeping this index & the menu item?
to @sbsmith86's point of if I want to see all comps I'm running?